File: FizzBuzz/FizzBuzz.cpp
#include <vector>
#include <iostream>
#include<string>

class Solution
{
public:
    std::vector<std::string> fizzBuzz(int n)
    {

        std::vector<std::string> results;

        for (int i = 1; i < n + 1; i++)
        {
            if ((i % 3 == 0) && (i % 5 == 0))
            {
                results.push_back("FizzBuzz");
            }
            else if (i % 3 == 0)
            {
                results.push_back("Fizz");
            }
            else if (i % 5 == 0)
            {
                results.push_back("Buzz");
            }
            else
            {
                results.push_back(std::to_string(i));
            }

            std::cout<< results[i];
        }

        return results;

    }
};

int main()
{
    Solution SolutionObj;
    SolutionObj.fizzBuzz(5);
    return 0;
}

File: FizzBuzz/FizzBuzz.java
package FizzBuzz;
// This is a simple Java program.

// FileName : "HelloWorld.java".

class HelloWorld {
    // Your program begins with a call to main().
    // Prints "Hello, World" to the terminal window.
    public static void main(String args[]) {
        for (int i = 1; i < 20; i++) {
            if ((i % 5 == 0) && (i % 3 == 0)) {
                System.out.println("FizzBuzz");
            } else if ((i % 5 == 0)) {
                System.out.println("Buzz");
            } else if ((i % 3 == 0)) {
                System.out.println("Fizz");
            } else {
                System.out.println(i);
            }
        }
    }
}